### DISCUSSION TITLE
Commented tests in nx.capacity_scaling()

### DISCUSSION THREAD

**Discussion by PurviChaurasia** [networkx/networkx#6481] [Q&A]
In reference to PR #6473, it was found that certain test cases in `algorithms/flow/tests/test_mincost.py` are commented out in the code base. Not commenting them also raises error in the test suite in some cases. Is there any specific reason for this? If so, wouldn't it be better practice to remove them?

**Comment by MridulS** [comment]
Well for that, we need to go combing through the git history and find when and why (hopefully) they were commented out. We should not remove them just because they are commented out. It's better to have a good idea about "why" before changing it :)

**Reply by PurviChaurasia** [reply]
That makes sense, shouldn't we comment out other such redundant tests? Because I faced a lot of problem while working on my PR. Just for future ease?
